### ISSUE TITLE
lldb support seems pretty broken in lldb-300.5.27.3

### ISSUE THREAD

**Issue by richo** [snare/voltron#17]
So far what I've deduced:

I can get voltron to work if I:
- start lldb
- load voltron
- start voltron
- attach a bunch of viewers
- manually call `voltron update`
- Do whatever I'm doing in lldb
- manually call `voltron update` every time I want the viewer to update.

I get a barf in `get_registers` that crashes lldb if it's run in a hook, but if you run `voltron update` from the debugger it dumps a stack trace. It looks like GetRegisters is returning an empty set.

tl;dr I'm happy to work on this but missing a bunch of domain knowledge about how lldb's interface works/how voltron is bound to it. If you're free, wanna give me a primer at some point?
